### PR TITLE
fix(chat): fix scroll down when expand image (Issue #1168)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

fix scroll down when expand image

Issues:

- Issue #1168

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/Chat.tsx [hunk 1]
```diff
@@ -738,69 +732,72 @@ export const ChatView = memo(() => {
...
-                                        }}
-                                        onRegenerate={
-                                          index === mergedMessages.length - 1 &&
-                                          showLastMessageRegenerate
-                                            ? onRegenerateMessage
-                                            : undefined
-                                        }
-                                        messagesLength={mergedMessages.length}
-                                      />
+                    <div ref={chatMessagesRef}>
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/Chat.tsx:735]: Do we need an additional div? Maybe we can put this ref in the div below?

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/Chat.tsx (hunk 1 above)
Comment (IlyaBondar) [reply apps/chat/src/components/Chat/Chat.tsx:735]: No, we can't - there is condition `mergedMessages?.length > 0`

File: apps/chat/src/components/Chat/MessageAttachment.tsx [hunk 2]
```diff
@@ -281,7 +296,10 @@ export const MessageAttachment = ({ attachment, isInner }: Props) => {
         </button>
       </div>
       {isOpenable && isOpened && (
-        <div className="relative mt-2 h-auto w-full overflow-hidden p-3 pt-4 text-sm duration-200">
+        <div
+          className="relative mt-2 h-auto w-full overflow-hidden p-3 pt-4 text-sm duration-200"
+          ref={anchorRef}
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/MessageAttachment.tsx:319]: As I know, React recommends to use only 1 `ref` for a particular element. Here `anchorRef` is applied for 2 elements

Comment (IlyaBondar) [reply apps/chat/src/components/Chat/MessageAttachment.tsx:319]: these 2 elements can NOT exist together and it's main idea for this anchor

Comment (IlyaBondar) [reply apps/chat/src/components/Chat/MessageAttachment.tsx:319]: Ohh. there it's mistake, but the comment is correct for `ChatMessageContent`

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1219.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**